examples/yew-tls/self_signed_certs/cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
The certificate seems to have expired 2022. I think for a certificate like this, having some expiry date in 2050 would work best :)